StreamChatUITestsAppUITests/MockServer/MockServerAttributes.swift

```diff
@@ -67,11 +69,18 @@ enum JSONKey {
     static let member = "member"
     static let id = "id"
     static let cooldown = "cooldown"
+    static let watcherCount = "watcher_count"
+    static let imageAction = "image_action"
```

Comment:
```suggestion
    static let attachmentAction = "attachment_action"
```

Reply:
Emm, but we send a request to the backend with the `image_action` param, not `attachment_action` 🤔

Reply:
Hmm, I didn't know we have this param somewhere.. Would you point me towards the endpoint? I might be missing some part around ephemeral messages

Reply:
Something like [this](https://github.com/GetStream/stream-chat-swift/blob/develop/TestTools/StreamChatTestTools/Fixtures/JSONs/Attachment/AttachmentPayloadGiphyWithActions.json#L10)?